src/components/side-panel/panel-header.js

```diff
@@ -123,17 +123,18 @@ const StyledPanelDropdown = styled.div`
 export const PanelAction = ({item, onClick}) => (
   <StyledPanelAction className="side-panel__panel-header__action"
     data-tip data-for={`${item.id}-action`} onClick={onClick}>
+    <p>Share</p>
```

Comment:
Actually we don't want every panel action to say: `Share`, so let's add a `label` property to item. and let this be
`{item.label ? <p>{item.label}</p> : null}`